app/utilities/Resources.java

```diff
@@ -0,0 +1,19 @@
+package utilities;
+
+/**
+ * Created by test on 02/02/16.
+ */
+public class Resources {
+    public static final String BAD_REQUEST = "Error in input data";
+    public static final String BAD_REQUEST_PASSWORD_MISMATCH = "Error in input data";
+    public static final String BAD_REQUEST_WRONG_CONFIRMATION_TOKEN = "Wrong confirmation token";
+    public static final String SERVER_NAME = "localhost:9000"; //"api.table4you" when deployed
+    public static final String VERSION = "v1.0";
+
+    public static boolean passwordCheck(UserHelper userHelper){
+        if (userHelper.password.equals(userHelper.passwordConfirmation))
+                return true;
+        return false;
+    }
+   // http://<server_name>/<version>/<url_path>?<arbitrary_params>.
+}
```

Comment:
3 comments for this simple method:

1. Please use brackets even if single statement, since it can lead to an error in further development process. This would also be good code formatting practice.

2. I wouldn't put this method in the same class as constants for response handling. This and similar methods should be encapsulated in a different class(es), grouped by some common behaviour, to avoid having too large classes.

3. This method's body could be written in a single line: 
`public static boolean passwordCheck(UserHelper userHelper){
     return userHelper.password.equals(userHelper.passwordConfirmation);
 } `